### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by snazy**
PR based on PR #32 

implement cleanup job

### CODE REVIEW

**Comment by BenWhitehead** [comment]
Please rebase on mesosphere:rewrite

File: cassandra-executor/src/main/java/io/mesosphere/mesos/frameworks/cassandra/jmx/NodeCleanupJob.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        return true;
+    }
+
+    @Override
+    public void startNextKeyspace() {
+        final String keyspace = super.nextKeyspace();
+        if (keyspace == null)
+            return;
+
+        new Thread("Cleanup " + keyspace) {
```
Comment (BenWhitehead) [inline cassandra-executor/src/main/java/io/mesosphere/mesos/frameworks/cassandra/jmx/NodeCleanupJob.java]: Please update executor to start and executor service and refactor this code to provide another runnable and can be scheduled when this job completes.

We should avoid starting untracked one off threads.

**Comment by BenWhitehead** [comment]
Please rebase on current `mesosphere:rewrite`

**Comment by snazy** [comment]
Please ignore this PR (superseded by newer PR)

**Pull request closed without merging**